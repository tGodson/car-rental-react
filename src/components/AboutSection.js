import React from 'react';
import { Row, Col } from 'react-bootstrap';
import ride from '../imgs/ride.jpg';

export default function AboutSection() {
  return (
    <section className="about-section">
      <h4 className="about-section__sub">Confort is Priceless</h4>
      <h2 className="about-section__primary pb-4">
        Comfortable rides from reliable and experienced people
      </h2>
      <Row className="mt-5">
        <Col sm={6}>
          <h4 className="about-section__left-title">
            A ride is always available for you 24/7
          </h4>
          <p className="about-section__desc">
            We are a car rental service available to you 24/7. We provide the most
            confortable and cheapest rides in the country. Hook us up for your trips and
            get the best confort can offer.
          </p>
          <p>
            <a href="#register" className="btn btn-oval">
              Learn more
            </a>
          </p>
        </Col>
        <Col sm={6}>
          <img src={ride} className="img-fluid" alt="ride" />
        </Col>
      </Row>
    </section>
  );
}
