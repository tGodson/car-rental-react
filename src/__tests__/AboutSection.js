import React from 'react';
import renderer from 'react-test-renderer';
import AboutSection from '../components/AboutSection';

it('renders AboutSection correctly', () => {
  const tree = renderer.create(<AboutSection />);
  expect(tree).toMatchSnapshot();
});
