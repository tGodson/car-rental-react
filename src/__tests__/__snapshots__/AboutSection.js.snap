// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutSection correctly 1`] = `
<section
  className="about-section"
>
  <h4
    className="about-section__sub"
  >
    Simple and fast way to rent
  </h4>
  <h2
    className="about-section__primary pb-4"
  >
    Comfortable ride from reliable people
  </h2>
  <div
    className="mt-5 row"
  >
    <div
      className="col-sm-6"
    >
      <h4
        className="about-section__left-title"
      >
        Ride Availability is 24/7
      </h4>
      <p
        className="about-section__desc"
      >
        You will have the best support to manage and collect every vehicle information about the ride. Full control will save you from any unforeseen situations and save you money and public transport related issues
      </p>
      <p>
        <a
          className="btn btn-oval"
          href="#register"
        >
          Learn more
        </a>
      </p>
    </div>
    <div
      className="col-sm-6"
    >
      <img
        alt="ride"
        className="img-fluid"
        src="ride.jpg"
      />
    </div>
  </div>
</section>
`;
